docs/make.jl

```diff
@@ -1,47 +1,28 @@
-using ImageMagick, Documenter, ESDL, ESDLPlots, Cairo
-
-
-newcubedir = mktempdir()
-ESDLdir(newcubedir)
-# Download Cube subset
-if !isempty(ESDL.ESDLDefaults.cubedir[])
-  c = S3Cube()
-  csa = c[
-    region = "South America",
-    var = ["country_mask","c_emissions","gross", "net_ecosystem", "air_temperature_2m", "terrestrial_ecosystem", "land_surface_temperature"],
-    time = 2003:2006
-  ]
-  saveCube(csa,"southamericacube", chunksize=(90,90,92,1))
-  ESDL.ESDLDefaults.cubedir[] = joinpath(newcubedir,"southamericacube")
-end
-
-exampledir = joinpath(@__DIR__,"src","examples")
-allex = map(readdir(exampledir)) do fname
-  n = splitext(fname)[1]
-  n => joinpath("examples",fname)
-end
-
+using Documenter, ESDL
 
 makedocs(
-    modules = [ESDL, ESDLPlots],
-    clean   = true,
+    modules  = [ESDL],
+    clean    = true,
+    doctest  = false,
     format   = Documenter.HTML(),
     sitename = "ESDL.jl",
-    authors = "Fabian Gans",
-    pages    = [ # Compat: `Any` for 0.4 compat
+    authors  = "Fabian Gans",
+    pages    = [ 
         "Home" => "index.md",
-        "Examples" => allex,
-        "Manual" => Any[
-            "cube_access.md",
-            "analysis.md",
-            "plotting.md",
-            "iotools.md"
+        "Examples" => Any[
+            "examples/time_mean.md",
+        ],
+        "API Documentation" => Any[
```

Comment:
```suggestion
        "How Tos" => Any[
```

For me, except for the index information these subpages are all How tos. I would move the Index page into the "Other functions" and rename that to "API Documentation". 
But I am not sure, whether it makes sense to go into the details here and maybe we should first decide which parts of the documentation should be moved to YAXArrays and what is going to stay here. 
Maybe you can have a subpage "YAXArrays" to have a place for all the moved documentation.

Reply:
Yes I also think index to API docs makes sense.
Some detail about the APIs is always good I feel, but the major documentation of them is definitively going in YAXArrays. 

Do you mean making YAXArrays itself as a subpackage of ESDL? Or just for the docs section?